src/mrack/transformers/openstack.py

```diff
@@ -32,8 +32,8 @@ class OpenStackTransformer(Transformer):
 
     async def init_provider(self):
         """Initialize associate provider."""
-        images = self.config["images"].values()
-        await self._provider.init(image_names=images)
+        images = self.config["images"]
+        await self._provider.init(images=images)
```

Comment:
This change is wrong. The original intention was to initialize an OpenStack provider with a list of images so that it can pre-load the info of images we care about and make the subsequent operations faster. With this change we are moving logic of knowing structure of provisioning config to provider. Translating metadata and provisioning config to host requirements is a job of transformer.